yoti_python_sdk/tests/mocks.py

```diff
@@ -11,7 +12,9 @@ def __init__(self, status_code, text, headers=None, content=None):
         if headers is None:
             headers = dict()
 
-        super(MockResponse, self).__init__(status_code, text, headers, content)
+        super(MockResponse, self).__init__(
+            status_code, text, CaseInsensitiveDict(headers), content
```

Comment:
I've used `CaseInsensitiveDict` here to mock the behaviour of _Requests_ case insensitive headers. It doesn't affect existing tests, but I thought it would be best for the mock headers to behave in the same way.